docs/campaign/quests/faucet.md

```diff
@@ -0,0 +1,66 @@
+# Faucet
+
+Faucets are a powerful tool for engaging and rewarding early-stage users within your community. With this in mind, the team at AirLyft has developed an intuitive faucet feature that allows communities to distribute blockchain-native assets to participants in a seamless and controlled manner.
+
+Imagine you are running a Web3 project that grants early access to users who have supported you since the beginning. You want to reward these users by allowing them to claim tokens, but only after verifying their eligibility. With AirLyft, you can create tasks that ensure only qualified participants can claim rewards.
+
+For example, you can check if their wallet address is on an approved whitelist. This streamlined process ensures that rewards go to genuine supporters while maintaining security and fairness within your community.
+
+Let's dive deep on how we can create these actions at AirLyft.
+
+- Login to AirLyft and create a campaign. On the Campaign page, click on Quests tab where you can find the Verified Faucet block as shown in the screenshot below.
+
+  ![](../../images/faucetinQuest.png)
+
+- Select an action from the list below that you would want the participants to perform.
+
+  - Select **Dotsama Faucet** to Transfer native dotsama asset to a participant .
+
+  - Select **EVM Faucet** to Transfer EVM gas token to a participant.
+
+    ![](../../images/faucetOptions.png)
+
+- Next, expand the 'Basic Details' option to enter the Title, Description, cFuel and XP. It is advisable to elaborate on the title and description so that participants have a much greater understanding of the task.
+
+- Then expand 'Task Condition and Recurrence' option to set the task frequency and task conditions.
+
+  - 'Task Frequency' determines how often users can complete a task. There are four options:
```

Comment:
Let's create a new page called "Task Condition & Recurrence" and then point all the pages to that page for recurrence & conditions instead of copying this on all pages